xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/AdminWikisTemplatesSheet.xml

```diff
@@ -41,25 +41,32 @@
 {{translation key="admin.wikis.templates.hint" /}}
 
 #set($columns = ['wikiprettyname', 'description', 'owner', 'membershipType'])
-#set($columnsProperties = {
-    'wikiprettyname' : { 'type' : 'text', 'link' : 'field' },
-    'description' : { 'type' : 'text' , 'size' : 50, 'html' : 'true' },
-    'owner' : { 'type' : 'text', 'size' : 10, 'html' : 'true' },
-    'membershipType' : { 'sortable': false, 'filterable': false}
-})
-#set($options = {
-   'className' : 'XWiki.XWikiServerClass',
-   'rowCount' : 10,
-   'resultPage' : 'WikiManager.WikisLiveTableResults',
-   'translationPrefix' : 'platform.wiki.browse.',
-   'extraParams': 'onlyTemplates=1',
-   'outputOnlyHtml': true
-})
 #if(!$isGuest)
   #set($discard = $columns.add('_actions'))
-  #set($discard = $columnsProperties.put('_actions', { 'actions' : ['edit', 'delete'] }))
 #end
-{{html clean="false"}}#livetable('wikis', $columns, $columnsProperties, $options){{/html}}
+
+#set ($liveDataConfig = {
+  'meta': {
+    'propertyDescriptors': [
+      { 'id': 'wikiprettyname', 'displayer': 'link' },
+      { 'id': 'membershipType', 'sortable': false, 'filterable': false }
+    ]
+  }
+})
+## TODO: onlyTemplates=1'?
+#set ($sourceParameters = $escapetool.url({
+  'className': 'XWiki.XWikiServerClass',
+  'resultPage': 'WikiManager.WikisLiveTableResults',
+  'translationPrefix': 'platform.wiki.browse.',
+  'onlyTemplates': '1'
```

Comment:
@mflorea all the rest is good but I didn't find a way to get rid of this source parameter.

Reply:
That's fine, it's a custom source parameter. See for instance https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-livetable/src/test/resources/liveTableConfigHelper/NotificationsFiltersPreferencesMacro.test#L22 . The items from ``extraParams`` that cannot be converted to a query (property) filter or to some other standard live data parameter must be kept as custom source parameters.

Reply:
ok, thanks for confirming, cleaning up the PR and merging then.